lightning/src/ln/channel.rs

```diff
@@ -784,6 +784,10 @@ pub(super) struct ChannelContext<Signer: ChannelSigner> {
 	pending_monitor_updates: Vec<PendingChannelMonitorUpdate>,
 }
 
+impl<Signer: ChannelSigner> ChannelContext<Signer> {
+
+}
+
```

Comment:
We'll just be using channel context to share behaviour too. It takes the more general ChannelSigner at least.